tests/foreman/api/test_contentmanagement.py

```diff
@@ -119,24 +119,28 @@ def test_positive_sync_kickstart_repo(self, default_sat):
 
         :BZ: 1687801
         """
-        org = entities.Organization().create()
-        product = entities.Product(organization=org).create()
-        repo = entities.Repository(
-            product=product, url=constants.repos.CUSTOM_KICKSTART_REPO
-        ).create()
-        repo.sync()
-        repo.download_policy = 'immediate'
-        repo = repo.update(['download_policy'])
-        call_entity_method_with_timeout(repo.sync, timeout=600)
+        rh_repo_id = enable_rhrepo_and_fetchid(
+            basearch='x86_64',
+            org_id=module_manifest_org.id,
+            product=constants.PRDS['rhel8'],
+            repo=constants.REPOS['rhel8_bos_ks']['name'],
+            reposet=constants.REPOSET['rhel8_bos_ks'],
+            releasever=None,
```

Comment:
```suggestion
            releasever='8.4',
```
This change should enable the repo successfully.  I have tried on my local.